packages/playground/src/components/members-list.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import {
+  Icon,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text,
+  Avatar,
+  AvatarImage,
+  AvatarFallback,
+  Badge
+} from '@harnessio/canary'
+import { getInitials } from '../utils/utils'
+
+interface MembersProps {
+  display_name: string
+  role: string
+  email: string
+  timestamp?: string
+  avatarUrl?: string
+}
+
+interface PageProps {
+  members: MembersProps[]
+}
+
+export const MembersList = ({ members }: PageProps) => {
+  return (
+    <Table variant="asStackedList" className="border-0">
+      <TableHeader>
+        <TableRow>
+          <TableHead className="text-primary">Name</TableHead>
+          <TableHead className="text-primary">Role</TableHead>
+          <TableHead className="text-primary">Email</TableHead>
+          {members[0]?.timestamp && <TableHead className="text-right text-primary">Date added</TableHead>}
+          <TableHead>
+            <></>
+          </TableHead>
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {members &&
+          members.map((member, index) => {
+            return (
+              <TableRow key={index}>
+                {/* NAME */}
+                <TableCell className="content-center my-6">
+                  <div className="flex items-center gap-4">
+                    <Avatar className="w-8 h-8">
+                      {member.avatarUrl && <AvatarImage src={member.avatarUrl} />}
+                      <AvatarFallback className="text-xs p-1 text-center">
+                        {getInitials(member.display_name, 2)}
```

Comment:
NIT: maybe define a variable for truncation len?

Reply:
fixed: using size instead